tests/pivots/pivot_calculated_measure.test.ts

```diff
@@ -1033,4 +1033,36 @@ describe("Pivot calculated measure", () => {
     expect(getEvaluatedCell(model, "E2").value).toEqual(40);
     expect(getEvaluatedCell(model, "E3").value).toEqual(60);
   });
+
+  test("formula with a domain not matching any data in the pivot", () => {
+    const grid = {
+      A1: "Customer",
+      B1: "Country",
+      C1: "Price",
+      A2: "Alice",
+      B2: "BE",
+      C2: "10",
+      A3: '=PIVOT.VALUE(1, "calculated", "Country", "BE", "Customer", "Bob")', // Missing Bob in BE
+      A4: '=PIVOT.VALUE(1, "calculated", "Country", "IN", "Customer", "Alice")', // Missing IN
+      A5: '=PIVOT.VALUE(1, "calculated", "Country", "IN", "Customer", "Bob")', // All missing
+      A6: '=PIVOT.VALUE(1, "calculated", "Country", "IN")', // aggregated value
+    };
+    const model = createModelFromGrid(grid);
+    const sheetId = model.getters.getActiveSheetId();
+    addPivot(model, "A1:C2", {
+      rows: [{ fieldName: "Country" }, { fieldName: "Customer" }],
+      measures: [
+        {
+          id: "calculated",
+          fieldName: "calculated",
+          aggregator: "sum",
+          computedBy: { formula: "='Country'&'Customer'", sheetId },
+        },
+      ],
+    });
+    expect(getEvaluatedCell(model, "A3").value).toEqual("BE");
```

Comment:
I don't think this result make much sense, no ? Shouldn't it be empty since BE+Bob does not exist in the pivot ?

Reply:
I'd say if it takes < 5 line of code it can be worth it. Otherwise, whatever

Reply:
Actually, fixing it "the easy way" (with `getPivotHeaderValueAndFormat(domain)` in 2 LoC) breaks a (tested) behavior that might be a valid use case

Here, there's no entry for "Customer=Bob, Category=Drink", even though Bob is part of the dataset. There's a world where you still want to get something from "Bob" or "Drink" with a calculated measure. (and it would be a breaking change anyway)
<img width="1864" height="796" alt="image" src="https://github.com/user-attachments/assets/49b11dcd-05a6-4545-a03a-b641a95a8198" />

Fixing it differently would require a new method for both spreadsheet and odoo pivot to check if values exists anywhere, no matter the combination in the domain. And that's not worth it IMO

Reply:
We can leave it as it then 🤷